pkg/network/setup/netstat_test.go

```diff
@@ -849,7 +851,10 @@ func newTestSetupWithVolatileCache() testSetup {
 }
 
 func newTestSetup() testSetup {
-	var cacheCreator tempCacheCreator
+	var (
+		cacheCreator      tempCacheCreator
+		clusterConfigurer = cConfigStub{dynamicPodInterfaceNamingEnabled: true}
```

Comment:
I would default to `off` as that is the default and use `on`  when you need (one test case?)

Reply:
Done